pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -143,6 +145,9 @@ let
       # ++ optional (versionRange "68" "72") ( githubPatch "<patch>" "0000000000000000000000000000000000000000000000000000000000000000" )
     ] ++ optionals (!stdenv.cc.isClang && (versionRange "71" "72")) [
       ( githubPatch "65be571f6ac2f7942b4df9e50b24da517f829eec" "1sqv0aba0mpdi4x4f21zdkxz2cf8ji55ffgbfcr88c5gcg0qn2jh" )
+    ] ++ optionals VAAPISupport [
+      ./patches/enable-vaapi-72.patch
+      ./patches/enable-mojo-video-decoders-by-default-72.patch
```

Comment:
Chromium 73 was released yesterday, so we should be able to drop the `./patches/enable-mojo-video-decoders-by-default-72.patch` needed only for 72 and require version 73 for enabled VAAPI support (default.nix:L81), as I imagine the number of people who need specifically 72 with VAAPI support should be negligible.
Also `enable-vaapi-72.patch` is not 72-specific, so you might want to remove that version number from the name?
There's also still a bunch of patch files in this PR that aren't actually used.